pages/blog/graphql-deep-dive-1.mdx

```diff
@@ -0,0 +1,354 @@
+  src="https://www.youtube.com/embed/Vq0ajno-zgw"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+<br />
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/LKQKn1oFXJU"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+**What are some of the GraphQL events/conferences to watch out for?**
+
+Since GraphQL was released, it garnered a huge interest in the community that a lot of conferences, events and meetups are held around the world keeping GraphQL as the main theme. Some of them are:
+
+- [**The GraphQL Summit**](https://summit.graphql.com/)
+- [**GraphQL Conf**](https://graphqlconf.org/)
+- [**Enterprise GraphQL**](https://hasura.io/enterprisegraphql/)
+- [**GraphQL Asia**](https://graphql.asia/)
+- [**GraphQL Galaxy**](https://graphqlgalaxy.com/)
+
+and there are more including meetups like [**these**](https://www.meetup.com/topics/graphql/) and [**these**](https://graphql.org/community/upcoming-events/). You can find most of the previous sessions recorded on Youtube if you search for it.
+
+**How can I contribute to GraphQL and its ecosystem?**
+
+Every bit of help really counts since GraphQL foundation is run by a set of volunteers and its all open source. You can
+
+- Write blogs like this to spread knowledge amongst the community
+- Host meetups, speak in conferences about your experience and evangelize your best way possible.
+- Contribute to the [**GraphQL spec**](https://github.com/graphql/graphql-spec) with your suggestions (Some suggestions may take years to implement even if it is good, so you may need to have a lot of patience for this)
+- Contribute to the ecosystem of tools leveraging GraphQL be it with documentation, tests, features, bug fixes, feedback and what not. It will definitely help.
+- Facing a challenge with GraphQL which has not been solved before? Build your own tooling and contribute it to the community
+- Or if you want to take it to the next level and want to align your entire organization to help the GraphQL foundation, [become its member](https://foundation.graphql.org/join/) and contribute.
+
+There are a lot of small ways in which you can give back. Small or big does not matter. Every contribution counts.
+
+**Are there some case studies which can actually help me in the implementation?**
+
+Sure. While I can’t list them all here, here are some:
+
+- [Netflix and GraphQL](https://netflixtechblog.com/how-netflix-scales-its-api-with-graphql-federation-part-1-ae3557c187e2)
+- [Airbnb and GraphQL](https://medium.com/airbnb-engineering/how-airbnb-is-moving-10x-faster-at-scale-with-graphql-and-apollo-aa4ec92d69e2)
+- [Github and GraphQL](https://www.graphql.com/articles/graphql-at-github)
+- [Twitter and GraphQL](https://about.sourcegraph.com/graphql/graphql-at-twitter/)
```

Comment:
Could you add this one as well? https://product.voxmedia.com/2020/11/2/21494865/to-federate-or-stitch-a-graphql-gateway-revisited

Reply:
Sure. Will add it, but I see the blog more as Schema stitching vs Federation rather than a case study - it may not belong in this section but will add it in another place.

Reply:
The main reason I feel that way is cause, it does not talk about how Vox uses Stitching, but just about stitching vs federation in general. While all the info is very useful, the context of the business is missing here.